File: TrackGenerator/track.c
//
// Created by malte on 09-Apr-20.
//

#include <stdio.h>

#include "track.h"

void Track_print(Track *track){
    printf(
            "Track( id=%ld, name='%s', planet='%s', seed=%ld, )",
            track->id, track->name, track->planet->name, track->seed
    );
}

File: UniverseUpdater/source/track.py
##The track object/DTO
class Track:

    def __init__(self, id, length, name, planet_id,):
        self.id = id
        self.length = length
        self.name = name
        self.planetId = planet_id



    def __str__(self):
        return f"Track( id={self.id},length='{self.length}', name='{self.name}', planetId={self.planetId} )"


